app/docs/src/.vitepress/config.ts

```diff
@@ -93,6 +93,10 @@ export default defineConfig({
             text: "Editing and Contributing Components",
             link: "/tutorials/editing-components-and-contributing",
           },
+          {
```

Comment:
This should be an explainer not a tutorial - it's a core set of changes - think how we do AWS Authentication